resources/views/help/admin/partials/division-checklist.md

```diff
@@ -4,6 +4,13 @@ Documentation of the various objects needed for Divisions. This page will genera
 adding a new division. Deviations for deleting a Division will also be noted.
 
 - [DNS Entry](#content-dns-entry)
+- [Tracker](#content-tracker)
+  - [Creating divisions in Admin CP](#content-creating-divisions-in-admin-cp)
```

Comment:
So, one of the reasons why the tracker stuff was put last was because a lot of the fields require things that need to be created in the forums first, or else the sync will fail. IE application form id, officer usergroup id, etc. So, we should take care to ensure these steps follow a logical order, or else it gets a lot more confusing

Reply:
That's fine, it was a ticket from Bear at Arch's direction so I set this up as per the ticket. If you guys want to handle this a different way then just let me know.

Reply:
@manimal-prime I think as long as it comes after the forums (but before the discord stuff) that should be fine if you wanna make that change